lib/multiplexer/proxyline.go

```diff
@@ -60,6 +60,38 @@ func (p *ProxyLine) String() string {
 	return fmt.Sprintf("PROXY %s %s %s %d %d\r\n", p.Protocol, p.Source.IP.String(), p.Destination.IP.String(), p.Source.Port, p.Destination.Port)
 }
 
+// Bytes returns on-the wire bytes representation of proxy line conforming to the proxy v2 protocol
+func (p *ProxyLine) Bytes() []byte {
+	var b []byte
+	b = append(b, proxyV2Prefix...)
+	b = append(b, (Version2<<4)|ProxyCommand)
+
+	switch p.Protocol {
+	case TCP4:
+		b = append(b, ProtocolTCP4)
+		b = append(b, 0, 12)
```

Comment:
What does 0 and 12 mean? Could you add a short comment or paste a reference?

Reply:
I've reworked this whole section to use structs instead, which avoids all these magic numbers